cypress/support/commands.js

```diff
@@ -22,7 +22,7 @@ const login = () => {
 
   cy.get('.login-card input[name=password]')
     .clear()
-    .type(Cypress.env('INTEGRATION_PASSWORD'));
+    .type('johnfoo+integration+password');
```

Comment:
Let's tweak this so that it's completely agnostic of any possible likeness to a real person. i.e. no names. As we're using `node-oidc-provider` now, this could literally just be the word `test`.